packages/box/lib/utils/unbox.ts

```diff
@@ -37,7 +44,18 @@ async function verifyURL(url: string) {
   }
 }
 
+async function verifyURL(url: string) {
+  if (url.startsWith("/")) {
+    return verifyLocalPath(url);
+  }
+  return verifyGithubURL(url);
```

Comment:
I'm not sure we should make this specific to github. I think these should work with services like bitbucket as well.